packages/vue/src/router.ts

```diff
@@ -51,23 +62,38 @@ export function vueRouterInstrumentation(router: VueRouter): VueRouterInstrument
         query: to.query,
       };
 
+      // Determine a name for the routing transaction and where that name came from
+      let transactionName: string = to.path;
+      let transactionSource: 'url' | 'custom' | 'route' = 'url';
```

Comment:
Totally optional but wdyt about extracting this to a type like in the [ReactRouterV3](https://github.com/getsentry/sentry-javascript/blob/6d98f6886c6c4a4d1af7f16f1afaf534c0ee8a24/packages/react/src/reactrouterv3.ts#L22) implementation? Just for consistency...

Reply:
Right, at the time of opening this I didn't see that we exported the TransactionSource type. Thanks!